tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+	"compilerOptions": {
+		"baseUrl": ".",
+		"rootDir": "./src",
+		"outDir": "./dist",
+		"target": "ES2022",
+		"module": "ES2022",
+		"allowJs": true,
+		"alwaysStrict": true,
+		"noEmitOnError": true,
+		"noImplicitAny": true,
+		"moduleResolution": "Node16",
+		"skipLibCheck": true,
+		"paths": {
+			"#src/*": ["src/*"]
+		}
+	},
+	"include": ["src/**/*"]
```

Comment:
Not sure if we have to explicitly put an `"exclude": [""]` to exclude `node_modules`, `dist`, and so on, I guess this is fine as is.